src/index.html

```diff
@@ -17,7 +17,7 @@ <h1 class="main-heading">Leaderboard</h1>
     <section class="scores-table">
       <div class="scores-top">
         <h3 class="scores-heading"></h3>
-        <a href="" class="refresh-btn">Refresh</a>
+        <button class="refresh-btn">Refresh</button>
```

Comment:
[REQUIRED] Kindly ensure that the refresh button is working as expected. The scores are not displayed when I click on it.

Reply:
Understood. Working on that right now.